### PR TITLE
Easy intro

### PR DESCRIPTION
**Pull request by rpspringuel**
I'm just getting started with this, but I want it to be available for comment as I work on it.

### CODE REVIEW

**Comment by rpspringuel** [comment]
As this tutorial gets longer, I'd like some suggestions for where to break it.  I'm afraid that if the tutorial is all on one page then the number of images could slow down the load time on slower connections.

File: tutorial/tutorial-gabc-01.html [hunk 1]
```diff
@@ -52,308 +52,149 @@
...
 
 <style type="text/css">
   .commandblock {font-family: courier; padding-left: 150px;}
   .commandline {font-family: courier;}
 </style>
-<h1>Gregorio and GABC Tutorial</h1>
+<h1>GABC Tutorial</h1>
 
-<p>The purpose of this tutorial is to demonstrate how to represent a Gregorian chant score in a gabc file and then include that file into a larger project which will produce a printed score (the method we recommend for everything but the simplest use cases).  For instructions on how to use the <code>\gabcsnippet</code> command for directly including short snippets of code in an existing <span class="tex">T<span class="epsilon">e</span>X</span> document, please <a href="./tutorial-snippet-01.html">see our other tutorial</a>.</p>
+<p>The purpose of this tutorial is to demonstrate how to represent a Gregorian chant score in gabc.  However, we are not going to assume a familiarity with the terminology used to describe such a score either in order to make this introduction easier to follow for those who are relatively new to chant more generally (and not just typesetting it).</p>
```
Comment (eschwab) [inline tutorial/tutorial-gabc-01.html]: I would drop the word "either" in the middle of the second sentence. It is confusing and seems to go with "in order".

File: tutorial/tutorial-gabc-01.html [hunk 2]
```diff
@@ -52,308 +52,149 @@
...
+    <dt><code>commentary: ;</code></dt>
+        <dd>This line is also an empty header field, but it's for specific information about the score.  In gabc parlance, the commentary is the text that appears just above the top line of the staff and is flushed right.  This is usually a scripture citation.  Our piece, however, doesn't have this line so we're going to delete this line.</dd>
+    <dt><code>annotation: ;</code></dt>
+        <dd>Like commentary, annotations refer to a very specific part of the score in gabc parlance.  In this case it's the text which goes above the initial.  Our score has two lines there, and fortunately gabc supports upto two annotation headers.</dd>
```
Comment (eschwab) [inline tutorial/tutorial-gabc-01.html]: upto -> up to

**Comment by rpspringuel** [comment]
Okay, the tutorial is finished content-wise.  Still need to edit it and fix some styling issues, but I'm fairly certain I've covered all the steps at this point.

**Comment by rpspringuel** [comment]
If there are no further comments, I'm going to merge this on Friday.

**Pull request merged**